src/request.ts

```diff
@@ -1040,6 +1040,13 @@ class DatastoreRequest {
           }
         }
 
+        if (entityObject.excludeFromIndexes) {
```

Comment:
We have the 'excludePathFromEntity' function in entity.ts. Could this new behavior be put into the same function? This is probably the most complex bit of code in our library, so anything we can do to make it more approachable is great.

Reply:
@stephenplusplus Hi, as suggested i implemented the new behavior in  function  excludePathFromEntity!